descqa/ColorDistribution.py

```diff
@@ -130,10 +136,11 @@ def run_on_single_catalog(self, catalog_instance, catalog_name, output_dir):
                    '{} < {}'.format(labels['redshift'], self.zhi)]
         data = catalog_instance.get_quantities(list(labels.values()), filters)
         data = {k: data[v] for k, v in labels.items()}
-
+        print(list(data.keys()))
+        
         # Color transformation
         color_trans = None
-        if self.color_transformation_q:
+        if self.color_transformation_q and filter_this != 'lsst':
```

Comment:
I don't think this second condition is needed, is it?  For filter that does not have corresponding transformation specified, transformation won't be applied.
```suggestion
        if self.color_transformation_q:
```

Reply:
I moved this check 2 lines down to make sure that the color transformation for DEEP2 data is not called in the case that LSST filters are being used (since this trasnformation doesn't exist)